pnpm-lock.yaml

```diff
@@ -8117,6 +7991,7 @@ packages:
 
   /debuglog@1.0.1:
     resolution: {integrity: sha512-syBZ+rnAK3EgMsH2aYEOLUW7mZSY9Gb+0wUMCFsZvcmiz+HigA0LOcq/HoQqVuGG+EKykunc7QG2bzrponfaSw==}
+    deprecated: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
```

Comment:
Looks like this is a dependency of [`readdir-scoped-modules@1.1.0`](https://www.npmjs.com/package/readdir-scoped-modules) which in itself is also deprecated.

> This functionality has been moved to @npmcli/fs

Reply:
appears to be mainly coming from webpack, so maybe we bump webpack and see if that fixes?

Reply:
So it seems! Looks like webpack is a dependency of docusaurus too, not just a direct `devDependency` for the cheatsheet.

Reply:
https://github.com/cursorless-dev/cursorless/issues/1748